setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+#
+# Bit of a hack, but lets expect jbs-installer directory to be cloned next to this one.
+if [ ! -d "$(dirname $0)/../jbs-installer" ]; then
+    echo "Clone jbs-installer"
+fi
+
+export OPENSHIFT_CLUSTER="$(kubectl get route -n openshift-console console -ojsonpath='{.status.ingress[0].host}' | sed 's/console-openshift-console.//')"
+echo "Domain is $OPENSHIFT_CLUSTER"
+kubectl config set-context --current --namespace=default
+
+echo -e "\033[0;32mRunning JBS demo kustomize...\033[0m"
+kustomize build . | kubectl apply -f -
+
+# Loop until the route is ready. Should be far quicker than waiting for the entire nexus startup.
+timeout=300
+endTime=$(( $(date +%s) + timeout ))
+until kubectl get --namespace=nexus routes.route.openshift.io nexus3 -o=jsonpath="{.status.ingress[0].host}" ; do
+    sleep 1
+    if [ $(date +%s) -gt $endTime ]; then
+        exit 1
+    fi
+done
```

Comment:
Actually I think I can replace this loop with
```
kubectl wait --namespace=nexus --for=jsonpath='{.status.ingress[0].host}' --timeout=480s routes.route.openshift.io nexus3 || exit 1
```
assuming we use kubectl >= 1.28